lib/src/git_subprocess.rs

```diff
@@ -52,6 +54,11 @@ pub enum GitSubprocessError {
     },
     #[error("Failed to wait for the git process")]
     Wait(std::io::Error),
+    #[error(
+        "Git does not recognize required option: {0} (note: required version is \
+         {MINIMUM_GIT_VERSION})"
```

Comment:
nit: since this is no longer the minimum required version, it might be better to say "minimum supported version" or something.